mirgecom/integrators/lsrk.py

```diff
@@ -34,6 +34,14 @@
 import numpy as np
 
 
+def lsrk4_step(state, t, dt, rhs):
+    """Call lsrk54_step with backwards-compatible interface."""
+    from warnings import warn
+    warn("Do not call lsrk4; it is now callled lsrk54_step. This function will "
+         "disappear April 1, 2021", DeprecationWarning, stacklevel=2)
```

Comment:
A month is probably a bit short. I usually do at least a few months.

Reply:
(fbe4769)